### PR TITLE
Utils documentation

### PR DESCRIPTION
**Pull request by OferLevi85**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: docs/zms_client.md [hunk 1]
```diff
@@ -220,7 +223,7 @@ following zms-cli command:
 
     $ zms-cli -d <domain> add-group-role <role> <member> [<member> ...]
 
-### Parameters
+<h3 id="parameters-1">Parameters</h3>
```
Comment (OferLevi85) [inline docs/zms_client.md:226]: Links didn't work for Parameters and examples as the id wasn't recognized. Solved it by using HTML headers (h3 is equivalent to ###)

File: docs/data_model.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ decentralized authorization, and how to set up role-based authorization.
 * Policy - a set of assertions that govern usage of resources
 * Assertion - a quadruple of &lt;effect, role, action, resource&gt;
 * Principal - an authenticated user or service that assumes one or more roles
-* User - a user identity that is authenticate by a particular authority like Bouncer
+* User - a user identity that is authenticate by a particular authority
```
Comment (OferLevi85) [inline docs/data_model.md:61]: Saw we removed references to Bouncer a couple of months ago so I removed it from here too

File: docs/zms_svctoken.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+based on given private key and service details.
+
+## Getting Software
+-------------------
+
+Download latest ZMS Service Token Client utility binary release from Bintray - click
+on the `Files` tab, choose the latest version directory and then
+download the `athenz-utils-<latest-version>-bin.tar.gz` file::
+
+[ ![Download](https://api.bintray.com/packages/yahoo/maven/athenz-utils/images/download.svg) ](https://bintray.com/yahoo/maven/athenz-utils/_latestVersion)
```
Comment (OferLevi85) [inline docs/zms_svctoken.md:20]: Noticed that all the utilities are part of the same package in bintray - athenz-utils.
I copied the "Getting Software" and "Preqreuisites" parts from the ZMS Util guide to all relevant docs.

File: docs/zts_roletoken.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+[ ![Download](https://api.bintray.com/packages/yahoo/maven/athenz-utils/images/download.svg) ](https://bintray.com/yahoo/maven/athenz-utils/_latestVersion)
+
+```shell
+$ tar xvfz athenz-utils-X.Y-bin.tar.gz
+```
+
+## Prerequisites
+----------------
+
+Before you can use the ZMS client utility, you need to have
```
Comment (havetisyan) [inline docs/zts_roletoken.md:28]: ZMS Client -> ZTS RoleToken

File: docs/zts_roletoken.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+$ zts-roletoken -domain <domain> [-role <role>] -svc-key-file <private-key-file> -svc-cert-file <service-cert-file> -zts https://zts-server.athenzcompany.com:4443/zts/v1 [-expire-time <expire-time-in-mins>]
+```
+
+### Using NToken from command line
+
+```
+$ zts-roletoken -domain <domain> [-role <role>] -ntoken <ntoken> -hdr Athenz-Principal-Auth -zts https://zts-server.athenzcompany.com:4443/zts/v1 [-expire-time <expire-time-in-mins>]
+```
+
+Only on-prem ZTS instance supports service ntokens and the `hdr` argument must be set
```
Comment (havetisyan) [inline docs/zts_roletoken.md:60]: remove line reference to on-prem ZTS

File: docs/zts_roletoken.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+Only on-prem ZTS instance supports service ntokens and the `hdr` argument must be set 
+to `Athenz-Principal-Auth`.
+The service identity ntoken can be obtained by using the [zms-svctoken](zms_svctoken.md) utility.
+
+### Using NToken from a given file
+
+```
+$ zts-roletoken -domain <domain> [-role <role>] -ntoken-file <ntoken-file> -hdr Athenz-Principal-Auth -zts https://zts-server.athenzcompany.com:4443/zts/v1 [-expire-time <expire-time-in-mins>]
+```
+Only on-prem ZTS instance supports service ntokens and the `hdr` argument must be set
```
Comment (havetisyan) [inline docs/zts_roletoken.md:69]: remove this line referencing on-prem ZTS

File: docs/zts_svccert.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
+* [Overview](#overview)
+* [Getting Software](#getting-software)
+* [Prerequisites](#prerequisites)
+* [Usage](#usage)
+
+## Overview
+----------
+
+ZTS TLS Certificate utility provides a service identity x.509 certificate
+based on registered service's ntoken in on-prem ZTS environment only.
```
Comment (havetisyan) [inline docs/zts_svccert.md:10]: remove the last section "in on-prem ZTS environment only"

File: docs/zts_svccert.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+[ ![Download](https://api.bintray.com/packages/yahoo/maven/athenz-utils/images/download.svg) ](https://bintray.com/yahoo/maven/athenz-utils/_latestVersion)
+
+```shell
+$ tar xvfz athenz-utils-X.Y-bin.tar.gz
+```
+
+## Prerequisites
+----------------
+
+Before you can use the ZMS client utility, you need to have
```
Comment (havetisyan) [inline docs/zts_svccert.md:28]: ZMS client -> ZTS ServiceCertificate

**Pull request merged**